src/RetryAnnotationTrait.php

```diff
@@ -121,43 +110,36 @@ private function getRetryDelayMethodAnnotation()
         } elseif (isset($annotations['class']['retryDelayMethod'][0])) {
             $delayAnnotation = $annotations['class']['retryDelayMethod'];
         } else {
-            return;
+            return null;
```

Comment:
Not strictly necessary but since the return type is `?array`, i.e. `array | null`, returning `null` is more appropriate as `return;` is closer to a `: void` return type

Reply:
Makes sense! And good catch. Thank you!